CacheBolt Could not deserialize message INTERNAL_ERROR

```
2018-02-10 11:42:24.424 o.o.w.t.c.CacheBolt [ERROR] Could not deserialize message {str={
    "timestamp": 1518195027758,
    "correlation_id": "system-request",
    "destination": "WFM_CACHE",
    "payload": {
        "error-type": "INTERNAL_ERROR",
        "error-message": "Server closed connection",
        "error-description": "Switches"
    },
    "clazz": "org.openkilda.messaging.error.ErrorMessage"
}}
com.fasterxml.jackson.databind.JsonMappingException: Unexpected token (END_OBJECT), expected FIELD_NAME: missing property 'clazz' that is to contain type id  (for class org.openkilda.messaging.error.ErrorData)
 at [Source: {
    "timestamp": 1518195027758,
    "correlation_id": "system-request",
    "destination": "WFM_CACHE",
    "payload": {
        "error-type": "INTERNAL_ERROR",
        "error-message": "Server closed connection",
        "error-description": "Switches"
    },
    "clazz": "org.openkilda.messaging.error.ErrorMessage"
}; line: 10, column: 14] (through reference chain: org.openkilda.messaging.error.ErrorMessage["payload"])
        at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:270) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.DeserializationContext.wrongTokenException(DeserializationContext.java:1376) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.DeserializationContext.reportWrongTokenException(DeserializationContext.java:1197) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer._deserializeTypedUsingDefaultImpl(AsPropertyTypeDeserializer.java:165) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:105) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeWithType(BeanDeserializerBase.java:1082) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:502) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:511) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:400) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1191) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:314) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:183) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:150) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer._deserializeTypedForId(AsPropertyTypeDeserializer.java:129) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.jsontype.impl.AsPropertyTypeDeserializer.deserializeTypedFromObject(AsPropertyTypeDeserializer.java:97) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.AbstractDeserializer.deserializeWithType(AbstractDeserializer.java:142) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.deser.impl.TypeWrappedDeserializer.deserialize(TypeWrappedDeserializer.java:63) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3798) ~[stormjar.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2842) ~[stormjar.jar:?]
        at org.openkilda.wfm.topology.cache.CacheBolt.doWork(CacheBolt.java:206) [stormjar.jar:?]
        at org.openkilda.wfm.topology.utils.AbstractTickStatefulBolt.execute(AbstractTickStatefulBolt.java:83) [stormjar.jar:?]
        at org.apache.storm.topology.StatefulBoltExecutor.doExecute(StatefulBoltExecutor.java:145) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.topology.StatefulBoltExecutor.handleTuple(StatefulBoltExecutor.java:137) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.topology.BaseStatefulBoltExecutor.execute(BaseStatefulBoltExecutor.java:78) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.daemon.executor$fn__9866$tuple_action_fn__9868.invoke(executor.clj:730) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.daemon.executor$mk_task_receiver$fn__9787.invoke(executor.clj:462) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.disruptor$clojure_handler$reify__9301.onEvent(disruptor.clj:40) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.utils.DisruptorQueue.consumeBatchToCursor(DisruptorQueue.java:472) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.utils.DisruptorQueue.consumeBatchWhenAvailable(DisruptorQueue.java:451) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.disruptor$consume_batch_when_available.invoke(disruptor.clj:73) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.daemon.executor$fn__9866$fn__9879$fn__9932.invoke(executor.clj:849) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at org.apache.storm.util$async_loop$fn__555.invoke(util.clj:484) [storm-core-1.1.0.2.6.2.0-205.jar:1.1.0.2.6.2.0-205]
        at clojure.lang.AFn.run(AFn.java:22) [clojure-1.7.0.jar:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_60]
```

--- a/services/topology-engine/queue-engine/topologylistener/message_utils.py
+++ b/services/topology-engine/queue-engine/topologylistener/message_utils.py
@@ -24,9 +24,11 @@ import config
 producer = KafkaProducer(bootstrap_servers=config.KAFKA_BOOTSTRAP_SERVERS)
 logger = logging.getLogger(__name__)
 
-MT_ERROR="org.openkilda.messaging.error.ErrorMessage"
-MT_COMMAND="org.openkilda.messaging.command.CommandMessage"
-MT_INFO="org.openkilda.messaging.info.InfoMessage"
+MT_ERROR = "org.openkilda.messaging.error.ErrorMessage"
+MT_COMMAND = "org.openkilda.messaging.command.CommandMessage"
+MT_INFO = "org.openkilda.messaging.info.InfoMessage"
+MT_ERROR_DATA = "org.openkilda.messaging.error.ErrorData"
+
 
 def get_timestamp():
     return int(round(time.time() * 1000))
@@ -179,7 +181,8 @@ def send_error_message(correlation_id, error_type, error_message,
     # TODO: Who calls this .. need to pass in the right TOPIC
     data = {"error-type": error_type,
             "error-message": error_message,
-            "error-description": error_description}
+            "error-description": error_description,
+            "clazz": MT_ERROR_DATA}
     send_to_topic(data, correlation_id, MT_ERROR, destination, topic)
 
 

